src/dependency/Pyegi/Pyegi/settings.py

```diff
@@ -2,62 +2,67 @@
 from PyQt6.QtCore import Qt, QCoreApplication
 import os
 import json
-from utils import set_style, Theme, get_settings
+import sys
+import typing
+from typing import Union
+from utils import set_style, Theme, get_settings, GLOBAL_PATHS
 
-utils_path = os.path.dirname(__file__)
-settings_file_path = utils_path + "/settings.json"
-themes_path = utils_path + "/Themes/"
+if typing.TYPE_CHECKING:
+    from main import Ui_MainWindow
 
 
-class Ui_SettingsWindow(object):
-    def setupUi(self, SettingsWindow, MainWindow=None):
+class Ui_SettingsWindow:
+    def setupUi(self, SettingsWindow, main_ui: Union["Ui_MainWindow", None] = None):
         SettingsWindow.setObjectName("SettingsWindow")
-        SettingsWindow.resize(308, 151)
+        SettingsWindow.resize(308, 171)
```

Comment:
I mentioned this before, but eventually we need to calculate these numbers based on screen resolution; probably